src/domain/collaboration/collaboration/collaboration.interface.ts

```diff
@@ -8,10 +8,6 @@ import { IInnovationFlow } from '../innovation-flow/innovation.flow.interface';
 
 @ObjectType('Collaboration')
 export abstract class ICollaboration extends IAuthorizable {
-  @Field(() => [ICallout], {
-    nullable: true,
-    description: 'List of callouts',
-  })
```

Comment:
Why is this removed?

Reply:
There is a dedicated field resolver, so this is a bug